### ISSUE TITLE
the hub - links with no accessible name

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#94]
https://rrchnm.org/the-hub/

ANDI shows 4 links without accessible names. It doesn't mention iframes, but I wouldn't count those out. 

WAVE also shows these images with missing alt text. They might be related. 
![Screen Shot 2020-10-06 at 12 28 58 PM](https://user-images.githubusercontent.com/618307/95230323-bdde0180-07cf-11eb-939c-66df1055fbb6.png)
![Screen Shot 2020-10-06 at 12 29 14 PM](https://user-images.githubusercontent.com/618307/95230326-be769800-07cf-11eb-95fb-782fae75aada.png)
![Screen Shot 2020-10-06 at 12 29 43 PM](https://user-images.githubusercontent.com/618307/95230330-bf0f2e80-07cf-11eb-873e-4fa0e357b0f9.png)
![Screen Shot 2020-10-06 at 12 29 51 PM](https://user-images.githubusercontent.com/618307/95230335-c0405b80-07cf-11eb-9d0f-013d871a26a7.png)

**Comment by kimisgold** [comment]
These images are from tweets external to the website. We can't go back to edit these tweets, but I've at least passed on the documentation for adding accessible descriptive text to future tweets that use images.
